.travis.yml

```diff
@@ -5,5 +5,5 @@ solution: WopiValidator.sln
 sudo: required
 dist: xenial
 script:
-  - dotnet build -c Release -f netcoreapp2.0
-  - dotnet test ./test/WopiValidator.Core.Tests/WopiValidator.Core.Tests.csproj -c Release -f netcoreapp2.0
+  - dotnet build -c Release -f net6.0
```

Comment:
Hi @mikeebowen, we are in the process of updating the versions shortly. Could you hold off on this PR? @borisf-msft fyi